### DISCUSSION TITLE
add separate applause sounds for each rank

### DISCUSSION THREAD

**Discussion by wsxdrfvyhn** [ppy/osu#13152] [Ideas]
basically getting a different applause sound every time you get a different rank, for example if you get an SS you will hear some loud clapping and cheering but if you get a D there will be lots of booing. would also be useful for skinners to add different voice clips of some fancy robotic speech synthesizer reading out your rank for example. the files would be called "applause_SS", "applause_S", "applause_A" and so on. there could also be a string in the skin.ini deciding whether or not to use all of those files for each rank, for example if it's set to 0 it will just use the normal "applause" file but if it's 1 it will use all these separate files for each rank.

**Comment by iSimpFor** [comment]
i think the applauding should start when you get a B grade(although it should be minimal)

the booing tells new players that's it's not a good idea to get those grades
C and D grades should have booing to encourage players to get higher grades

**Comment by peppy** [comment]
The recent implementation (#10571) added two levels of applause, but we may expand on this going forward.
